### ISSUE TITLE
Filtering small chromosomes option?

### ISSUE THREAD

**Issue by GlyphDataServices** [igvteam/igv.js#1110]
Hi- I see in genome.js ~line 480 small chromosomes are filtered if < median contig length/50. Unfortunately this results in the removal of GRCh38 mitochondrion and all the alt/unplaced contigs from the chromosomes menu, which I want to include.  

Could this be set as a config switch instead, whether or not to filter small reference contigs? 

`wgChromosomes = Object.values(genome.chromosomes).filter(chr => chr.bpLength > threshold)`

I set threshold to 0 as a temporary workaround. I can't imagine it would be much of an issue even if there are hundreds of additional contigs in that menu. One can also build a reference genome that excludes small unplaced/alt contigs if they're not wanted. 

Thanks

**Comment by jrobinso** [comment]
Hi, 
Yes we can add an option.  However this does not remove the data, you can still get to it by typing the chromosome name in the search box.  The filter controls 2 things  (1) the "whole genome" view, and (2) the pulldown menu.   Hundreds would not be a problem.   Several thousand as in the human genome might be.   In any event we can add an option,  I'll leave this open until we do.

**Comment by GlyphDataServices** [comment]
I noticed disabling this adds a lot of tiny un-clickable sections to the "all" view and depending where they are in the assembly this could be confusing.

in my GRCh38.p13 RefSeq I have 192 contigs including MT and alt/unplaced.  I think it depends on whose assembly and which patch. I agree though, thousands of menu items and clickable regions can cause issues.

Thanks for the tip on searching. That mitigates this somewhat since I can load regions with other interactions.

**Comment by jrobinso** [comment]
Yes, that is the main point of the filtering (the "all" view).     There probably should be separate options for the menu and "all" view,  or something other widget than a pull down list for the menu that could handle thousands of entries.   In any event leave this open as a reminder to do something about it.   I will not be able to get to it right away.

**Comment by jrobinso** [comment]
@GlyphDataServices I've just found time to look at this, and I wonder what version of igv.js you are using because the filtering of small chromosomes from the pulldown was removed in march of 2019.

**Comment by GlyphDataServices** [comment]
Looks like 2.2.2. I'll update. Thanks.
